src/main/java/com/uid2/client/InputUtil.java

```diff
@@ -0,0 +1,127 @@
+package com.uid2.client;
+
+import java.util.Base64;
+
+public class InputUtil { //from https://github.com/IABTechLab/uid2-operator/blob/master/src/main/java/com/uid2/operator/service/InputUtil.java
+  /**
+   * @param phoneNumber a phone number in any format
+   * @return whether phoneNumber is normalized, which is a requirement for {@link TokenGenerateInput#fromPhone}
+   */
+  public static boolean isPhoneNumberNormalized(String phoneNumber) {
+    //from https://github.com/IABTechLab/uid2-operator/blob/14de3733e72f72adf1d9af7091dee03ea9cdb5b2/src/main/java/com/uid2/operator/service/InputUtil.java#L80
```

Comment:
Should we move this logic to shared and use it here?

Reply:
Yes, although not sure we want to add a dependency on Shared for this SDK yet. 
Maybe revisit when we make Sharing changes?

Reply:
Let's not make client SDK depend on shared for now. Shared is intended for server side.

Reply:
In future, maybe we split out a dependency for components that both client and server need.
(btw "client SDK" can be confused with the javascript client SDK, naming is hard :)

Reply:
I've been calling the javascript client SDK just the "javascript SDK" or "JS SDK" - maybe we need better naming for these things.